gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.1-all.zip
+networkTimeout=10000
```

Comment:
That's needed because AGP 7.4.x needs Gradle 7.5:
https://developer.android.com/studio/releases/gradle-plugin

I've bumped to the latest stable of Gradle then